MyHashMap.java

```diff
@@ -0,0 +1,85 @@
+// Time Complexity : finding element is O(1)
```

Comment:
// Time Complexity : O(n) where n is the length of the LL formed at any index of the array
// O(1) for finding in array * O(n) for finding in LL = O(n)